src/main/java/com/gmail/yuramitryahin/model/CinemaHall.java

```diff
@@ -5,7 +5,19 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.Table;
-
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
```

Comment:
don't this  annotation yet but i think they will work )

Reply:
everything works, this annotations make life easier)

Reply:
@YuraMitriakhin see what does the `@Data` annotation)

Reply:
read, but we still have to declare constructors annotation because @Data have only @RequiredArgsConstructor, did I understand correctly?